dl_binaries.sh

```diff
@@ -49,7 +45,7 @@ echo "$targets" | while read -r target; do
   mkdir -p "binaries/$version/$target"
   path=$version/$target
   echo "downloading $path"
-  curl -fsS "https://update.tabnine.com/bundles/$path/TabNine.zip" -o "binaries/$path/TabNine.zip" ||
+  curl -fsS "$TABNINE_UPDATE_SERVICE/bundles/$path/TabNine.zip" -o "binaries/$path/TabNine.zip" ||
```

Comment:
I suggest to add an option to pass the URL here - then it need to be `url/update` + `...rest of path`